docs/tactics.md

```diff
@@ -1424,3 +1424,21 @@ by simp_rw [set.image_subset_iff, set.subset_def]
 ```
 
 Lemmas passed to `simp_rw` must be expressions that are valid arguments to `simp`.
+
+## rename'
+
+A variant of the standard `rename` tactic with the following improvements:
```

Comment:
`rename` isn't documented here because it isn't in mathlib. We don't have a clear policy on including core tactics in this list. But for the time being, could you expand this a little so it doesn't reference something undocumented?

Reply:
Thanks for the review! I've modified the docs so that the comparison with `rename` is less prominent.